Doors/doors.js

```diff
@@ -0,0 +1,61 @@
+const { select } = require("@inquirer/prompts");
+const { renderGame } = require("../bin/textRPG");
+
+function door1() {
```

Comment:
I think you should replace the name of the function door1() by treasureDoor() and the door2() by dangerDoor()

@FCarvalhal what do you think?